auth-web/package.json

```diff
@@ -55,10 +55,10 @@
     "sinon": "^6.3.5",
     "stylus": "^0.54.7",
     "stylus-loader": "^3.0.2",
-    "ts-jest": "^23.0.0",
-    "typescript": "^3.2.1",
+    "ts-jest": "23.0.0",
+    "typescript": "~3.5.3",
```

Comment:
Not going up to 3.6 or 3.7?

Reply:
let me give a try with 3.7 ..if it works , its good to go with 3.7 ... this is the version which vue cli proposes.. But 3.7 is great..Let me give a try

Reply:
I think we used it already.so will just work.. I will just try an optional chaining feature just to see how it goes

Reply:
Awesome